packages/ui-components/src/components/text-field/text-field.types.ts

```diff
@@ -0,0 +1,20 @@
+export enum EValidationState {
+	NONE,
```

Comment:
We usually use lowercase for enum values.

Reply:
And more important, this is not being used. I think this was related to the `formControl` dependency.

Reply:
Yes, it is really! I will remove it.